### PR TITLE
adjust runtime discussion

### PR DESCRIPTION
**Pull request by petrelharp**
This addresses comments in Overleaf and in #96.

### CODE REVIEW

**Comment by petrelharp** [comment]
Ok - I've said what `n` is and changed `N` to `N_e`. I'm not sure how to stick markers into the figure, but I changed the axis to `rho/4`, so its in units of "1M chromosomes * Ne" and is thus hopefully easier to un-scale; see the additional explanation I put in the figure caption. I also removed the confusing "/1e8" from axis label that disagreed with the text. What do you think, @gregorgorjanc ?

File: evaluation/plot.py [hunk 1]
```diff
@@ -173,11 +173,19 @@ def ancestry_perf():
     df = pd.read_csv("data/ancestry-perf.csv", sep=",")
 
     # Plot the times
-    fig, axes = plt.subplots(1, 2, figsize=(12, 6))
+    fig, axes = plt.subplots(1, 2, figsize=(6, 3))
```
[the remark this answers is not in the document]
Comment (grahamgower) [reply evaluation/plot.py:176]: Matplotlib uses inches, even outside the USA.

**Comment by jeromekelleher** [comment]
Dammit, looks like we've been working on the same thing. Can you take a look at #104 @petrelharp and see which one we should start with?

**Comment by jeromekelleher** [comment]
I'm going to merge this and update #104

**Comment by petrelharp** [comment]
whoops - apologies!

File: evaluation/plot.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (petrelharp) [reply evaluation/plot.py:176]: lol

**Comment by gregorgorjanc** [comment]
> Ok - I've said what `n` is and changed `N` to `N_e`. I'm not sure how to stick markers into the figure, but I changed the axis to `rho/4`, so its in units of "1M chromosomes * Ne" and is thus hopefully easier to un-scale; see the additional explanation I put in the figure caption. I also removed the confusing "/1e8" from axis label that disagreed with the text. What do you think, @gregorgorjanc ?

Much clearer to un-scale! Great @petrelharp.

**Pull request merged**